src/creature.cpp

```diff
@@ -203,16 +203,19 @@ static units::volume size_to_volume( creature_size size_class )
     // e.g. max tiny size is 7500, max small size is 46250, we return
     // 46250+7500 / 2 - 1_ml = 26875_ml - 1ml
     // This is still stupid and both of these functions should be merged into one single source of truth.
+
+    //Updated Calculation Because this one seems off
```

Comment:
I'm not sure how you've determined it "seems off", but your math is just plain incorrect.

Max tiny size is 7500ml, max small size is 46250, the difference is (alternatively) represented as 38750, therefore the midpoint between the two is 7500 + (38750 / 2) = 7500 + 19375 = 26875, as calculated. 26875 is 19375 away from 7500, and 19375 away from 46250.

Reply:
I incorrectly expressed myself my point was more of taking half of max volume as an average volume for size as it looked a bit better to me (and for Huge I took it min value as it is as far as i can see the only option for mutated characters to still be able to use vehicles

Reply:
If you belive that this is inappropriate solution i will change it to previous values.